client/apps/edit/components/content/sections/related_articles/index.jsx

```diff
@@ -0,0 +1,151 @@
+import async from 'async'
+import request from 'superagent'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import { compact, pluck, without } from 'underscore'
+import { data as sd } from 'sharify'
+import { EditArticleCard } from './components/edit_article_card'
+import { RelatedArticlesInput } from './components/related_articles_input'
+import { ArticleCard } from '@artsy/reaction-force/dist/Components/Publishing/Series/ArticleCard'
+import DraggableList from '../../../../../../components/drag_drop/index.coffee'
+
+export class RelatedArticles extends Component {
+  static propTypes = {
+    article: PropTypes.object.isRequired,
+    color: PropTypes.string,
+    onChange: PropTypes.func
+  }
+
+  state = {
+    relatedArticles: [],
+    loading: true
+  }
+
+  componentWillMount = () => {
+    this.fetchArticles()
+  }
+
+  fetchArticles = () => {
+    const { related_article_ids } = this.props.article.attributes
+    let relatedArticles = []
+
+    if (related_article_ids && related_article_ids.length) {
+      return async.each(related_article_ids, (id, cb) => {
+        return request
+          .get(`${sd.API_URL}/articles/${id}`)
```

Comment:
Ooh we should be using GraphQL! It removes the need for using `async.each` as well. It might be worth merging #1484 first to get it working otherwise you can always pass in `ids` to the normal articles resource.

Even with this REST way, you can fetch with `${sd.API_URL}/articles?ids=[123]` to avoid that `async.each`.